components/raftstore/src/coprocessor/properties.rs

```diff
@@ -76,6 +81,10 @@ impl MvccProperties {
         res.num_rows = props.decode_u64(PROP_NUM_ROWS)?;
         res.num_puts = props.decode_u64(PROP_NUM_PUTS)?;
         res.num_versions = props.decode_u64(PROP_NUM_VERSIONS)?;
+        // To be compatible with old versions.
+        res.num_deletes = props
+            .decode_u64(PROP_NUM_DELETES)
+            .unwrap_or_else(|_| res.num_versions - res.num_puts);
```

Comment:
If they are equal, then why add the metrics explicitly?

Reply:
I guess that it is just to be compatible with old versions....

Reply:
They are not equal. Actually,
```
num_version = num_puts + num_rollback + num_lock + num_delete
```

Reply:
why not just put it to the last when encoding